packaging/src/main/distributions/text/config/sink-pattern-relationship-quickstart.properties

```diff
@@ -1,5 +1,5 @@
 name=Neo4jSinkCypher
-topics=people
+topics=knows
```

Comment:
Does this change affect docs?

Reply:
Nope, we don't need to change anything on docs